database/migrations/2019_06_21_204239_comments.php

```diff
@@ -0,0 +1,151 @@
+
+        Schema::table("events", function ($table) {
+            $table->dropColumn('opmerkingen');
+        });
+
+        DB::statement("
+            INSERT INTO comments (text, entity_type, entity_id, is_secret, user_id)
+                 SELECT opmerkingen, 'App\\\\Location', id, false, 0
+                   FROM locations
+                  WHERE opmerkingen is not null
+        ");
+        Schema::table("locations", function ($table) {
+            $table->dropColumn('opmerkingen');
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+
+        Schema::table("members", function ($table) {
+            $table->string('opmerkingen')->nullable();
+            $table->string('opmerkingen_admin')->nullable();
+            $table->string('opmerkingen_geheim')->nullable();
+        });
+        DB::statement("
+            UPDATE members m
+              LEFT JOIN comments uc on uc.entity_id = m.id and uc.entity_type = 'App\\\\Member' AND uc.user_id = 0 and uc.is_secret = false
+              LEFT JOIN comments ac on ac.entity_id = m.id and ac.entity_type = 'App\\\\Member' AND ac.user_id = 0 and ac.is_secret = true and left(ac.text, 1) != '©'
+              LEFT JOIN comments sc on sc.entity_id = m.id and sc.entity_type = 'App\\\\Member' AND sc.user_id = 0 and sc.is_secret = true and left(sc.text, 1) = '©'
+               SET m.opmerkingen = uc.text, m.opmerkingen_admin = ac.text
+        ");
+
+        Schema::table("participants", function ($table) {
+            $table->string('opmerkingen')->nullable();
+            $table->string('opmerkingen_admin')->nullable();
+        });
+        DB::statement("
+            UPDATE participants p
+              LEFT JOIN comments uc on uc.entity_id = p.id and uc.entity_type = 'App\\\\Participant' AND uc.user_id = 0 and uc.is_secret = false
+              LEFT JOIN comments ac on ac.entity_id = p.id and ac.entity_type = 'App\\\\Participant' AND ac.user_id = 0 and uc.is_secret = true
+               SET p.opmerkingen = uc.text, p.opmerkingen_admin = ac.text
+        ");
+
+        Schema::table("events", function ($table) {
+            $table->string('opmerkingen')->nullable();
+        });
+        DB::statement("
+            UPDATE events e
+              LEFT JOIN comments uc on uc.entity_id = e.id and uc.entity_type = 'App\\\\Event' AND uc.user_id = 0 and uc.is_secret = false
+               SET e.opmerkingen = uc.text
+        ");
+
+        Schema::table("locations", function ($table) {
+            $table->string('opmerkingen')->nullable();
+        });
+        DB::statement("
+            UPDATE locations l
+              LEFT JOIN comments uc on uc.entity_id = l.id and uc.entity_type = 'App\\\\Location' AND uc.user_id = 0 and uc.is_secret = false
+               SET l.opmerkingen = uc.text
+        ");
+
+        Schema::dropIfExists('comments');
+        DB::statement("DELETE FROM users WHERE id = 1");
```

Comment:
Ik neem aan dat dit statement is om de SYS user te verwijderen? Zo ja dan moet dit `id = 0` zijn toch?

Reply:
Jaa dat denk ik ook, oeps. zal ik aanpassen